kw

```diff
@@ -193,21 +197,21 @@ function kw()
       ;;
 
     clear-cache)
-        . "$KW_LIB_DIR/mk.sh" --source-only
+        include "$KW_LIB_DIR/mk.sh"
 
         cleanup
       ;;
     # Subsystems support
     drm)
       (
-        . "$KW_PLUGINS_DIR/subsystems/drm/drm.sh" --source-only
+        include "$KW_LIB_DIR/subsystems/drm/drm.sh"
```

Comment:
Why did you replace `$KW_PLUGINS_DIR` with `$KW_LIB_DIR`?

Reply:
uou, nice catch. I was probably blindly making the adaption and didn´t realise the path for drm was different. Sorry about that